### ISSUE TITLE
Please remove c*nt - misogynist slur

### ISSUE THREAD

**Issue by EuphorbiaHorrida** [freebee-game/enable#1451]
Yes, I know it’s “still a word,” unlike most of you who will come to complain about this thread I’ve uttered it on stage multiple times in front of a live audience - but all that is beside the point. Exercising basic internet hygiene is a good thing, and nothing good can come of publishing a word game where the only way to beat it (i.e. find all the words) will frequently require typing an offensive slur and pressing enter.

**Comment by mouldilocks** [comment]
It is a body part.

**Comment by EuphorbiaHorrida** [comment]
> It is a body part.

It is called a vulva.

**Comment by BigRobPage** [comment]
It actually has other meanings too,  its mostly only considered particularly offensive in North America, due to the specific misogynistic way it is often used here. That said I was sort of surprised it was an accepted word, that said obviously people are still going to type it anyway when they are guessing words (we all seem to have)  and for what its worth, Its not a valid scrabble word.

**Comment by mouldilocks** [comment]
Not quite:  the vulva is only part of it.
Patricia

**Comment by EuphorbiaHorrida** [comment]
> Not quite: the vulva is only part of it. Patricia
> […](#)
> On 1. Nov 2023, at 17:23, EuphorbiaHorrida ***@***.***> wrote: ﻿ It is a body part. It is called a vulva. — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.

Well aware, not the point.

**Comment by mouldilocks** [comment]
So it isn’t called a vulva!
BigRob:  it’s offensive in the UK too but not misogynistic because it’s used to refer to men too.

**Comment by EuphorbiaHorrida** [comment]
> It actually has other meanings too, its mostly only considered particularly offensive in North America, due to the specific misogynistic way it is often used here. That said I was sort of surprised it was an accepted word, that said obviously people are still going to type it anyway when they are guessing words (we all seem to have) and for what its worth, Its not a valid scrabble word.

I opened this issue knowing about other meanings and its connotations outside of North America. I didn’t bring it up because it’s not the point. Every time someone suggests removing offensive slurs, these same excuses come up (often from the same people) as if the existence of benign connotations entirely render their notoriously known offensive definitions nonexistent.

As if I should be completely unbothered if a doctor says it to me while my feet are in stirrups. Unreal.

**Comment by mouldilocks** [comment]
Lots of words can be offensive words.  Would you remove bastard, turd & twat for example?
Patricia

**Comment by EuphorbiaHorrida** [comment]
> So it isn’t called a vulva! BigRob: it’s offensive in the UK too but not misogynistic because it’s used to refer to men too.

No, it is called a vulva. Every body part has constituent and contiguous parts, that doesn’t mean I’m wrong about it being called a vulva. Furthermore, calling men a misogynist slur does not change its notoriously known etymology and established meaning. That is not how language works.

**Comment by EuphorbiaHorrida** [comment]
> Lots of words can be offensive words. Would you remove bastard, turd & twat for example? Patricia
> […](#)
> On 1. Nov 2023, at 18:29, EuphorbiaHorrida ***@***.***> wrote: ﻿ It actually has other meanings too, its mostly only considered particularly offensive in North America, due to the specific misogynistic way it is often used here. That said I was sort of surprised it was an accepted word, that said obviously people are still going to type it anyway when they are guessing words (we all seem to have) and for what its worth, Its not a valid scrabble word. I opened this issue knowing about other meanings and its connotations outside of North America. I didn’t bring it up because it’s not the point. Every time someone suggests removing offensive slurs, these same excuses come up (often from the same people) as if the existence of benign connotations entirely render their notoriously known offensive definitions nonexistent. As if I should be completely unbothered if a doctor says it to me while my feet are in stirrups. Unreal. — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.

Your question is a distraction and I am not going to answer it, other than to say I don’t make removal requests lightly and I specifically target slurs because they harm the dignity of the human beings impacted.

The reason anybody requests that a word be removed from a word game is to try and make the world a kinder place one tiny step at a time, which is all any individual human can be expected to do. Since the request is directed at the developer and not you personally, I do not expect you to turn around and show support. However, I can’t help noticing you are almost always the first and loudest person who steps in and tries to put a stop to it, and that is not something where you can dress up your actions with innocent intentions and claim you are doing the right thing.

**Comment by ibara** [comment]
I do remove words that are not valid Scrabble words. And indeed this one is not. So it will go next time I get around to updating the dictionary.

In the future, to avoid the mob, you can just email me directly. I suppose this is a general announcement to everyone, not OP specifically. No one else has the ability to modify the dictionary other than me.
